bpo-37284: Add note to sys.implementation doc (#14328)
python/cpython: Doc/library/sys.rst

   .. versionadded:: 3.3

   .. note::

>       All proposals for new required attributes must go through the normal PEP

Reviewer: A slight tweak to make this more clear and direct:

```suggestion
      The addition of new required attributes must go through the normal PEP
```
